components/molecules/UniversalBenefitCard.js

```diff
@@ -96,11 +99,20 @@ export default function UniversalBenefitCard(props) {
               behavior: 'smooth',
               block: 'start',
             })
+            setBtnCaption(
+              isOpen
```

Comment:
_My comment didn't post_
As far as I know, to meet a11y we can't change a heading based on a state. (the caption is used as a h5 I believe)

Reply:
Ok, how else should we deal with changing the text depending on the benefit and status?

Do we know exactly what the key/value pairs will be from the benefits objects returned from our api?

Reply:
Sorry what I meant by "state" was open or closed, as in the user did something and the text in the header changed, that's not allowed. So when it's open, we can't change the text to "view less" (that's also insufficient at heading text, as it's not descriptive of the content within the section)

So all I'm suggesting is we remove the condition on `isOpen` and only use the text  reflective of program+status for both open and closed states (this is a pretty common thing on web pages).

I've mentioned the issue with Breanna last week, just have to confirm with design/content.

Reply:
Right, so that should be brought up to the design team. Let's discuss this with them on stand up.